lib/servers/Pulser2/pulse_sequences/pulse_sequence.py

```diff
@@ -12,7 +12,7 @@ class pulse_sequence(object):
 	required_subsequences = []
 	replaced_parameters = {}
 	
-	def __init__(self, parameter_dict, start = WithUnit(0, 's')):
+	def __init__(self, parameter_dict, start = WithUnit(1.0, 'us')):
```

Comment:
Best for now is likely what you have implemented.

Please just note in the docstring of init that the pulser hardware won't start with 0, and that is why the default is 1.0us.